.github/workflows/publish-stage.yaml

```diff
@@ -20,8 +20,6 @@ jobs:
           node-version: 16
       - name: Install Dependencies
         run: npm ci
-      - name: Disable Indexing
-        run: "sed -i 's/noIndex: false/noIndex: true/g' docusaurus.config.js"
```

Comment:
....while previously, we were changing _in staging_ from `noIndex: false` to `noIndex: true`.